### PR TITLE
feat: add recordPicker stubs

### PR DESCRIPTION
**Pull request by thomasFoulon**
Introducing lightning-record-picker stub for testing.

### CODE REVIEW

File: src/lightning-stubs/recordPicker/recordPicker.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    @api label;
+    @api matchingInfo;
+    @api objectApiName;
+    @api placeholder;
+    @api required;
+    @api value;
+    @api variant;
+    @api checkValidity() {}
+    @api reportValidity() {}
+    @api setCustomValidity() {}
```
Comment (RamiNoufal) [inline src/lightning-stubs/recordPicker/recordPicker.js:21]: fix: add 
- `focus` method
- `blur` method

**Review by RamiNoufal — CHANGES_REQUESTED** [review]

File: src/lightning-stubs/recordPicker/recordPicker.js (hunk 1 above)
Comment (thomasFoulon) [reply src/lightning-stubs/recordPicker/recordPicker.js:21]: Good point, those methods were not on the stub in the lightning-components repo 🤦

Comment (thomasFoulon) [reply src/lightning-stubs/recordPicker/recordPicker.js:21]: Update: `focus` and `blur` methods are not added to the stub when re-generating stubs on the lightning-components repo. It seems they are not necessary (they're not on other components either).

Comment (RamiNoufal) [reply src/lightning-stubs/recordPicker/recordPicker.js:21]: Native method are not needed it seems according to LBC [https://github.com/salesforce-experience-platform-emu/lightning-components/blob/ma[…]/lightning-components-stubs/scripts/generate-lightning-mocks.js](https://github.com/salesforce-experience-platform-emu/lightning-components/blob/master/lightning-components-stubs/scripts/generate-lightning-mocks.js#L193)

**Pull request merged**